src/main/resources/application.properties

```diff
@@ -36,3 +36,9 @@ spring.mail.properties.mail.smtp.auth=true
 spring.mail.properties.mail.smtp.starttls.enable=true
 spring.mail.properties.mail.smtp.starttls.required=true
 spring.mail.properties.mail.smtp.timeout=8000
+
+#-------------------------
+# Token Expiration Settings
+#-------------------------
+app.auth.tokenExpirationMsec = ${TOKEN_EXPIRATION_MSEC}
+app.auth.refreshTokenExpirationMsec= ${REFRESH_TOKEN_EXPIRATION_MSEC}
```

Comment:
What happened if these two parameters not set?
Does it make sense to set some default values?